src/dfx/src/commands/pull.rs

```diff
@@ -114,3 +130,87 @@ async fn fetch_deps_to_pull(
         },
     }
 }
+
+#[context("Failed while download wasm of canister {canister_id}.")]
```

Comment:
or `Failed while downloading`
```suggestion
#[context("Failed to download wasm of canister {canister_id}.")]
```